### PR TITLE
Updated configs

### PR DESCRIPTION
**Pull request by jturnbull**
@maxpeterson - let's discuss a way to make this repo private

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1 @@
+Copyright 2014 Incuna Ltd
```
Comment (meshy) [inline LICENCE]: EOL @ EOF

**Comment by meshy** [comment]
Looks good other than nitpick

**Comment by jturnbull** [comment]
_shakes fist_

**Comment by meshy** [comment]
haha

Blank line at EOF. `:D`

**Pull request merged**